libs/util/data-fetcher/src/parsers/excel.ts

```diff
@@ -22,25 +22,6 @@ export function parseExcel(buffer: ArrayBuffer): Promise<{
   })
 }
 
-export class ExcelDataset extends BaseDataset {
-  private parseResult_ = this.fetchAsBuffer().then(parseExcel)
-  private propertiesInfo_ = this.parseResult_.then(
-    (result) => result.properties
-  )
-  private datasetInfo_ = this.parseResult_.then(
-    (result) =>
-      ({
-        itemsCount: result.items.length,
-      } as DatasetInfo)
-  )
-  get properties(): Promise<PropertyInfo[]> {
-    return this.propertiesInfo_
-  }
-
-  get info(): Promise<DatasetInfo> {
-    return this.datasetInfo_
-  }
-  readAll(): Promise<DataItem[]> {
-    return this.parseResult_.then((result) => result.items)
-  }
+export class ExcelDataset extends BaseFileDataset {
+  parseResult_ = this.fetchAsBuffer().then(parseExcel)
```

Comment:
I think clearer to embed that in a method.
`fetchData` idk

Reply:
done in the latest version